src/librustc_resolve/lib.rs

```diff
@@ -3049,8 +3037,8 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
         }
     }
 
-    fn with_no_errors<T, F>(&mut self, f: F) -> T where
-        F: FnOnce(&mut Resolver) -> T,
+    fn with_no_errors<T, F>(&mut self, f: F) -> T
+        where F: FnOnce(&mut Resolver) -> T
```

Comment:
It scales to:
```
fn with_no_errors<T, F>(&mut self, f: F) -> T where
    where F: FnOnce(&mut Resolver) -> T,
          T: MoreBoundsHere,
```

I find that easier to scan because you see the where keyword when scanning down the lhs